js/app/controllers/content-product.ng

```diff
@@ -138,7 +138,158 @@ mkdir my-new-bucket</pre>
+      <cmdlet-ref name="Set-BqDataset"></cmdlet-ref> to manipulate cloud resources. This cmdlet also
+      handles adding and removing labels with <code>-SetLabel</code> and <code>-ClearLabel</code>.
+    </p>
+    <pre>
+# Updates the Name field of $dataset.
+$dataset.Name = "A Different Name"
+$dataset = Set-BqDataset "my_dataset"
 
+# Adds the labels 'test' and 'other' to $dataset.
+$dataset = Set-BqDataset "my_dataset" -SetLabel @{"test"="one";"other"="two"}
+    </pre>
+    <p>
+      Datasets can be deleted by the <cmdlet-ref name="Remove-BqDataset"></cmdlet-ref> cmdlet. This 
+      cmdlet supports ShouldProcess (the <code>-WhatIf</code> parameter) and will prompt for user 
+      confirmation before deleting a non-empty Dataset. This safeguard can be bypassed with the 
+      <code>-Force</code> parameter when scripting.  
+    </p>
+    <pre>
+# Deletes $dataset.
+$dataset | Remove-BqDataset -Force
+    </pre>
+    
+    <h2>Table</h2>
+    <p>
+      Each Dataset has a number of <em>Tables</em> to hold data. Tables are created with the 
+      <cmdlet-ref name="New-BqTable"></cmdlet-ref> cmdlet by passing in a TableId and a Dataset 
+      where the table will reside. The Dataset can be passed with a Dataset object or with the
+      <code>-DatasetId</code> parameter. <cmdlet-ref name="Get-BqTable"></cmdlet-ref> and 
+      <cmdlet-ref name="Set-BqTable"></cmdlet-ref> work the same way as the Get- and Set- cmdlets above.
+    </p>
+    <pre>
+# Creates a new table in the dataset from above.
+$table = $dataset | New-BqTable "my_table" -Description "Log data"
+
+# Gets a reference object for "my_dataset:my_table".
+$table = Get-BqTable "my_table" -DatasetId "my_dataset"
+
+# Modifies the Name attribute of my_table.
+$table.Name = "Logs"
+$table = $table | Set-BqTable
+    </pre>
+    <p>
+      Tables can be deleted by the <cmdlet-ref name="Remove-BqTable"></cmdlet-ref> cmdlet. This 
+      cmdlet supports ShouldProcess (the <code>-WhatIf</code> parameter) and will prompt for user 
+      confirmation before deleting a Table that contains data. This safeguard can be bypassed with 
+      the <code>-Force</code> parameter.  
+    </p>
+    <pre>
+# Deletes $table.
+$table | Remove-BqTable -Force
+    </pre>
+    
+    <h2>Schema</h2>
+    <p>
+      Tables need <em>Schemas</em> to describe the format of the data they contain. Schemas are created with
+      the <cmdlet-ref name="New-BqSchema"></cmdlet-ref> and <cmdlet-ref name="Set-BqSchema"></cmdlet-ref>
+      cmdlets. New-BqSchema can take the formats for rows as parameters directly or as a JSON array of 
+      row descriptions. The results of New-BqSchema are always passed into Set-BqSchema which can either output
+      a Schema object or assign the Schema to an existing Table.  
+    </p>
+    <pre>
+# Assigns a Schema to $table
+$table = Get-BqTable "my_table" -DatasetId "my_dataset"
+New-BqSchema "Author" "STRING" | New-BqSchema "Copyright" "STRING" | 
```

Comment:
Probably because it ends with `|` and just "figures it out". The F# dev in me wants to put that on the next line, which would probably cause the PowerShell parser to choke... Just keep it as-is I guess. Thanks for following up.